lib/graphql/batch.rb

```diff
@@ -12,20 +12,22 @@ class NestedError < StandardError; end
     def self.batch
       raise NestedError if GraphQL::Batch::Executor.current
       begin
-        GraphQL::Batch::Executor.current = GraphQL::Batch::Executor.new
+        GraphQL::Batch::Executor.current = GraphQL::Batch::Executor.new # TODO
```

Comment:
Changing this isn't important for our purposes, so you could just remove the `# TODO` comment.